Configuration/Skimming/interface/DisappearingMuonsSkimming.h

```diff
@@ -22,19 +22,21 @@
 #include <memory>
 
 // user include filter
-#include "FWCore/Utilities/interface/EDGetToken.h"
-#include "FWCore/Framework/interface/one/EDFilter.h"
-#include "FWCore/ParameterSet/interface/ParameterSet.h"
-#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
-#include "DataFormats/TrackReco/interface/Track.h"
-#include "TrackingTools/TransientTrack/interface/TransientTrack.h"
-#include "DataFormats/VertexReco/interface/Vertex.h"
 #include "DataFormats/Common/interface/TriggerResults.h"
 #include "DataFormats/EcalRecHit/interface/EcalRecHitCollections.h"
```

Comment:
```suggestion
```
It does not seem to be needed

Reply:
as a matter of fact it's needed instead, see:
https://github.com/cms-sw/cmssw/blob/7a3a15df8d7e392f2e1394ca2120e5c5f2e6512c/Configuration/Skimming/interface/DisappearingMuonsSkimming.h#L71-L72